packages/mds-repository/repository.ts

```diff
@@ -15,14 +15,18 @@
  */
 
 import { Connection } from 'typeorm'
-import { ConnectionManager, ConnectionManagerOptions, ConnectionMode } from './connection'
+import { ConnectionManager, ConnectionManagerOptions, ConnectionMode, ConnectionManagerCliOptions } from './connection'
 import { CreateRepositoryMigration } from './migration'
 
 type RepositoryMethod<TMethod> = (connect: (mode: ConnectionMode) => Promise<Connection>) => TMethod
 
 export const CreateRepositoryMethod: <TMethod>(
   method: RepositoryMethod<TMethod>
-) => RepositoryMethod<TMethod> = method => method
+) => RepositoryMethod<TMethod> = method => {
+  // eslint-disable-next-line no-console
+  console.warn('CreateRepositoryMethod is deprecated. Extend ReadWriteRepository instead.')
+  return method
+}
```

Comment:
`log.warn` instead of `console.warn` and probably add a TODO to remove this method at some point soon

Reply:
The deprecation messages and the methods themselves will go away shortly in a subsequent PR